modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/BadgePillFilter/index.jsx

```diff
@@ -8,9 +8,19 @@
  * permissions and limitations under the License, including but not limited to
  * distribution rights of the Software.
  */
+import {useMemo} from 'react';
+import i18n from '~/common/I18n';
 import {Button} from '../../../../common/components';
-
 const BadgePillFilter = ({filterName, filterValue, onClick}) => {
+	const getTranslatedValues = useMemo(() => {
```

Comment:
Is not a function anymore

```suggestion
	const translatedValues = useMemo(() => {
```